src/test/obj_fragmentation2/obj_fragmentation2.c

```diff
@@ -125,7 +125,7 @@ allocate_objects(PMEMobjpool *pop, size_t size_min, size_t size_max)
 static void
 delete_objects(PMEMobjpool *pop, float pct)
 {
-	size_t nfree = (float)nobjects * pct;
+	float nfree = (float)nobjects * pct;
```

Comment:
You can't do that. Think about what will happen in the loop later if the result of 9 * 10 is not 90, but 90.00000000001.